frontend/test/acceptance/JoinSupporterSpec.scala

```diff
@@ -1,122 +0,0 @@
-package acceptance
```

Comment:
no point in testing joining as that functionality was already removed!
Deleting the whole acceptance directory seemed to break the tests (they hang with a configuration issue) so I left it for now.